### DISCUSSION TITLE
Would you pay for Official PB hosting subscription?

### DISCUSSION THREAD

**Discussion by miguelgargallo** [pocketbase/pocketbase#2075] [Q&A]
Would you pay for Official PB hosting subscription? 
This will help users to ensure SSL and Security hosting by the Official team like MongoDB, MindsDB... and also exclusive features from beta possible version.

(I tried to make a Poll, but I was unable to do so.)

**Comment by denisgurev** [comment]
I asked for this in the past in https://github.com/pocketbase/pocketbase/discussions/432.

I would pay if there was a cloud service like Supabase. @ganigeorgiev Please at least consider it after v1.

I've tested pockethost.io and it is close to what I want but it doesn't allow custom migrations (at least the last time I checked it didn't) and was very slow. I'm currently self hosting PocketBase on Hetzner after the suggestion in the above discussion and to be honest the PocketBase setup alone is nothing complicated (only 1 binary and service file), but configuring the domain, dns records, finding a working mail (I'm using mailjet after an unsuccessful attempt with amazon) was very difficult for me and I remember spending whole day troubleshooting it.
